app/components/sendToken/index.js

```diff
@@ -117,233 +161,558 @@ export const Manual = props => {
       // if (selectedToken.balance !== undefined && parseFloat(fromAmount) > parseFloat(selectedToken.balance)) {
       //   setFromAmount(selectedToken.balance)
       // }
-      if (parseFloat(tokenAllowance) < parseFloat(fromAmount) && selectedToken.symbol !== 'BNB') {
-        setUserHasApproveToken(false)
+      if (
+        parseFloat(tokenAllowance) < parseFloat(fromAmount) &&
+        selectedToken.symbol !== 'BNB'
+      ) {
+        setUserHasApproveToken(false);
       }
-      if(fromAmount !== ""){
-        console.log(parseFloat(fromAmount) ,selectedToken.balance)
-        if(parseFloat(fromAmount) > parseFloat(selectedToken.balance)){
-          setInsufficientBalanceButton(true)
-        }else{
-          setInsufficientBalanceButton(false)
+      if (fromAmount !== '') {
+        console.log(parseFloat(fromAmount), selectedToken.balance);
+        if (parseFloat(fromAmount) > parseFloat(selectedToken.balance)) {
+          setInsufficientBalanceButton(true);
+        } else {
+          setInsufficientBalanceButton(false);
         }
       }
     }
-
   }, [fromAmount, amountIn]);
-  const importToken = (token) => {
-    token.available = true
-    token.imported = true
-    onCloseModal()
+
+  useEffect(() => {
+    const checkLiquidityPair = async () => {
+      const factory = await SmartFactory();
+      const fromPath = ethers.utils.getAddress(selectedToken.address);
+      const toPath = ethers.utils.getAddress(selectedToToken.address);
+      const LPAddress = await factory.getPair(toPath, fromPath);
+
+      const RGPBUSDAddress = await factory.getPair(
+        BSCTestnetTokens.RGP,
+        BSCTestnetTokens.BUSD,
+      );
+
+      const RGPBNBAddress = await factory.getPair(
+        BSCTestnetTokens.RGP,
```

Comment:
This `BSCTestnetTokens.RGP` is the testnet address of RGP. The mainnet address is entirely different which means this only works on the testnet. That's why I said you don't need to export the testnet address because there is a place on that same page where addresses are conditionally exported based on chain Id